.github/actions/upload-test-artifacts/action.yml

```diff
@@ -43,14 +43,14 @@ runs:
         FILE_SUFFIX: ${{ inputs.file-suffix }}
       run: |
         # Remove any previous test reports if they exist
-        rm -f usage-log-*.zip
```

Comment:
Why does this change?

Reply:
it doesn't have to change, i just think it's a more appropriate name